libvips/foreign/tiff2vips.c

```diff
@@ -1687,6 +1688,29 @@ rtiff_autorotate( Rtiff *rtiff, VipsImage *in, VipsImage **out )
 	return( 0 );
 }
 
+/* Unpremultiply associative alpha, if any.
+ */
+static int
+rtiff_unpremultiply( Rtiff *rtiff, VipsImage *in, VipsImage **out )
+{
+	if( rtiff->header.premultiplied == 1 ) {
+		VipsImage *x = vips_image_new();
```

Comment:
`vips_unpremultiply()` will allocate the output image, you don't need to make one here. In fact, making one will leak when `vips_unpremultiply()` writes to `x`.